ci/Dockerfile-envoy-alpine

```diff
@@ -2,7 +2,7 @@ FROM frolvlad/alpine-glibc:alpine-3.12_glibc-2.31
 RUN mkdir -p /etc/envoy
 
 ADD configs/envoyproxy_io_proxy.yaml /etc/envoy/envoy.yaml
-RUN apk add --no-cache shadow su-exec \
+RUN apk add --no-cache shadow \
```

Comment:
dont we want to continue using the apk version for the alpine build ?

Reply:
my bad, I forgot to add
```
ADD ${TARGETPLATFORM}/build_release/su-exec /usr/local/bin/
```
to this file to keep things consistent, let me do that right away

Reply:
or do you prefer that I just let this be the way it was ?

Reply:
personally - i think better the way it was.

we only added it in the other container image because it wasnt available in the distro

Reply:
sg, will revert this portion